src/book/03-guides/03-hardware/04-DARwIn-OP2-guide.mdx

```diff
@@ -0,0 +1,184 @@
+---
+section: Guides
+chapter: Hardware
+title: DARwIn OP2 Robot Restoration and Calibration Guide
+description: How to use the provided instructions for setting up, restoring, and calibrating the DARwIn OP2 robot system.
+slug: /guides/hardware/DARwIn-OP2-guide
+authors:
+  - Corah Oliver (@yippyy)
+---
+
+## DARwIn OP2 Guide
+
```

Comment:
I'd put something here about the Darwin robots, and maybe a photo too of them. Something about how we used to use them in the competition and now we use them for outreach activities.

Reply:
Also the change from OP to OP2